code/modules/organs/limbs.dm

```diff
@@ -666,7 +666,10 @@ Note that amputating the affected organ does in fact remove the infection from t
 	if(body_part == CHEST)
 		return FALSE
 
-	set_limb_flags(LIMB_DESTROYED)
+	if(amputation)
+		set_limb_flags(LIMB_AMPUTATED & LIMB_DESTROYED)
```

Comment:
this is the same as
set_limb_flags(0) do you mean |

Reply:
i sure do whoops